src/services/categories-service.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+export const fetchCategories = async (dispatch) => {
+    try {
+        const { data: { categories } } = await axios({
+            method: "GET",
+            url: "/api/categories",
```

Comment:
can replace this by `axios.get('/api/categories')`
It would looks much simpler to read